stroeer/fragment/stage/v1/stage_service.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+import "stroeer/page/section/v1/section_page.proto";
+
+package stroeer.fragment.stage.v1;
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/fragment/stage/v1;stage";
```

Comment:
please add generated go code to the PR

Reply:
done and https://github.com/stroeer/tapir/issues/229